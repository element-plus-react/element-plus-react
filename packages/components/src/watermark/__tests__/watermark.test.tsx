import React from "react";
import { describe, expect, it } from "vitest";
import Watermark from "..";

const AXIOM = "Rem is the best girl";

describe("Watermark", () => {
  it("create", () => {
    const wrapper = <Watermark className="watermark">{AXIOM}</Watermark>;

    expect(wrapper.props.className).toContain("watermark");
    expect(wrapper).toMatchSnapshot();
  });

  it("slots", () => {
    const wrapper = <Watermark>{AXIOM}</Watermark>;

    expect(wrapper.props.children).toContain(AXIOM);
  });
});
