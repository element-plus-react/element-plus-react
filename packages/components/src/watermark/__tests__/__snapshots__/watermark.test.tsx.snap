// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Watermark > create 1`] = `
<Watermark
  className="watermark"
>
  Rem is the best girl
</Watermark>
`;
